README.md

```diff
@@ -35,13 +35,13 @@ Alternatively, you can compile from source code by following the below steps:
 - Open it in Visual Studio, select the plugin you want to enable and select `publish` \(compile it using Release configuration\)
 - Create the Plugins folder in neo-cli / neo-gui (where the binary file is located, such as `/neo-cli/bin/Release/netcoreapp3.0/Plugins`)
 - Copy the .dll and the folder with the configuration files into the `Plugins` folder.
-  - Remarkably, you should put the dependency of the plugin in the `Plugins` folder as well. For example, since the `RpcServer` has the package reference on the `Microsoft.AspNetCore.ResponseCompression`, so the corresponding dll file should be put together with the plugin.
+  - Remarkably, you should put the dependency of the plugin in the `Plugins` folder as well. For example, since the `OracleService` has the package reference on the `Google.Protobuf.dll`, so the corresponding dll file should be put together with the plugin.
```

Comment:
We don't use google protobuf

Reply:
I think that they are needed for oracle, no?
Our building script copies theses ones.
```
    cp -ri $ORIGIN_PATH/Neo.FileStorage.API.dll $DEST_FOLDER
    cp -ri $ORIGIN_PATH/Grpc.Net.Common.dll $DEST_FOLDER
    cp -ri $ORIGIN_PATH/Grpc.Net.ClientFactory.dll $DEST_FOLDER
    cp -ri $ORIGIN_PATH/Grpc.Net.Client.dll $DEST_FOLDER
    cp -ri $ORIGIN_PATH/Grpc.Core.Api.dll $DEST_FOLDER
    cp -ri $ORIGIN_PATH/Grpc.AspNetCore.Server.dll $DEST_FOLDER
    cp -ri $ORIGIN_PATH/Grpc.AspNetCore.Server.ClientFactory.dll $DEST_FOLDER
    cp -ri $ORIGIN_PATH/Google.Protobuf.dll $DEST_FOLDER
    ```